Well, I seem to be a bit stuck since I'm not very experienced with cmake.  Its giving the following error when building with MSVC:
```
C:/projects/drltrace/build/bin/drltracelib.exp
drltrace.obj : error LNK2019: unresolved external symbol "int __cdecl fast_strcmp(char *,unsigned int,char *,unsigned int)" (?fast_strcmp@@YAHPADI0I@Z) referenced in function "void __cdecl lib_entry(void *,void * *)" (?lib_entry@@YAXPAXPAPAX@Z) [C:\projects\drltrace\build\drltracelib.vcxproj]
C:\projects\drltrace\build\bin\drltracelib.dll : fatal error LNK1120: 1 unresolved externals
```

It would appear that this is a problem because `drltracelib` isn't linked against `drltrace_utils.cpp`.  However, I see this in `CMakeLists.txt:293`:
```
set(srcs
    drltrace.cpp
    drltrace_libcalls.cpp
    drltrace_options.cpp
    drltrace_utils.cpp)
```

I'm not sure why the MSVC linker is failing.

--- a/drltrace_src/drltrace_utils.cpp
+++ b/drltrace_src/drltrace_utils.cpp
@@ -51,7 +51,7 @@ uint op_ignore_asserts = false;
 /* A faster(?) version of strcmp(), since strcmp() does extra string
  * comparison we don't need (we just need an equality test).  Returns
  * 0 when strings are equal, otherwise returns non-zero. */
-inline int
+int
 fast_strcmp(char *s1, size_t s1_len, char *s2, size_t s2_len) {
   if (s1_len != s2_len)
     return -1;

